spiceaidocs/docs/reference/task_history.md

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Task History'
+sidebar_label: 'Task History'
+pagination_prev: 'reference/index'
+pagination_next: null
+sidebar_position: 4
+---
+
+The Spice runtime stores information about completed tasks in the `spice.runtime.task_history` table. Each task executed has a row in this table, and by default the data is retained for 8 hours. Use a `SELECT` query to return information about each task as shown in this example:
+
+```sql
+SELECT
+  *
+FROM
+  spice.runtime.task_history
+LIMIT
+  100;
+```
+
+## Task Types
+
+The following top-level task types are currently recorded:
+
+| Task Type         | Description                        |
+| ----------------- | ---------------------------------- |
+| `sql_query`       | SQL Query                          |
+| `nsql_query`      | Natural Language to SQL Query      |
+| `ai_chat`         | AI Chat Completion                 |
+| `vector_search`   | Vector Search                      |
+| `accelerated_refresh` | Accelerated Table Refresh      |
+| `text_embed`      | Text Embedding                     |
+
+## Configuration
+
+Set the following parameters in the `runtime.task_history` section of your `config.yaml` file to configure task history:
+
+- `enabled`: Enable or disable task history.
+- `retention_period`: The duration for which task history data is retained.
+- `retention_check_interval`: The interval at which the task history retention is checked.
+- `captured_output`: The level of output captured for tasks. `none` or `truncated`.
```

Comment:
Provide defaults and examples.

Reply:
Added defaults, examples are already below